src/libraries/System.Text.Json/src/System/Text/Json/BitStack.cs

```diff
@@ -5,6 +5,7 @@
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
 
+#nullable enable
```

Comment:
why is this directive required? nullable is already set in `System.Text.Json.csproj`, shouldn't that also cover this file?

Reply:
It is covered for the project, it was also referenced in another Test project and was failing the test build